lib/contentful_model/version.rb

```diff
@@ -1,3 +1,3 @@
 module ContentfulModel
-  VERSION = "0.2.0"
+  VERSION = '0.2.0'.freeze
```

Comment:
Shouldn't the version number increase? Since you are renaming something, I was hoping for a change in here too.